pkg/sdk/go/things_test.go

```diff
@@ -35,7 +35,8 @@ const (
 var (
 	metadata   = map[string]interface{}{"meta": "data"}
 	metadata2  = map[string]interface{}{"meta": "data2"}
-	thing      = sdk.Thing{ID: "001", Name: "test_device", Metadata: metadata}
+	thing1     = sdk.Thing{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "test1", Metadata: metadata}
```

Comment:
Same here, probably `th1` would be a more consistent name

Reply:
Like above, moving them to th1, th2 for now.